db/schema.rb

```diff
@@ -235,6 +235,7 @@
     t.integer "team_id", null: false
     t.boolean "registration_open", default: false
     t.string "urn"
+    t.boolean "permission_to_observe_required"
```

Comment:
Could read slightly clearer if `involved_in_user_research_observation`
Very wordy though.

Reply:
Just my €0.02 ... I think `observed_session_agreed` -> `agreed_to_user_research_observation` gets us good value, more than this one.